frameworks/JavaScript/nodejs/hello.js

```diff
@@ -19,9 +19,9 @@ var http = require('http')
   , url = require('url')
   , async = require('async')
   , mongoose = require('mongoose')
-  , conn = mongoose.connect('mongodb://localhost/hello_world')
+  , conn = mongoose.connect('mongodb://127.0.0.1/hello_world')
```

Comment:
This file looks like `localhost` was replaced with `127.0.0.1` accidentally as well.